config/initializers/session_store.rb

```diff
@@ -0,0 +1,11 @@
+# Be sure to restart your server when you modify this file.
+
+if Rails.env.development?
+  Rails.application.config.session_store :cookie_store, key: '_commitchange_session'
+elsif Rails.env.staging? || Rails.env.production?
```

Comment:
Any reason not to use cookie store for all levels?

Reply:
Yes, there is! The details are [here](https://github.com/CommitChange/tix/issues/4007). I discussed on private SfG Slack and we decided it was better to not use Cookie Store.

I have mixed feelings on it.